docs/src/guides/inscriptions.md

```diff
@@ -0,0 +1,209 @@
+Ordinal Inscription Guide
+=========================
+
+Individual satoshis can be inscribed with arbitrary content, creating
+Bitcoin-native digital artifacts that can be held in a Bitcoin wallet and
+transferred using Bitcoin transactions. Inscriptions are as durable, immutable,
+secure, and decentralized as the Bitcoin blockchain itself.
+
+Working with inscriptions requires a Bitcoin full node, to give you a view of
+the current state of the Bitcoin blockchain, and a wallet that can make
+ordinal-aware transactions that inscribe satoshis with content and transfer
+individual satoshis using ordinal theory.
+
+Bitcoin Core provides both a Bitcoin full node and Bitcoin wallet. However, the
+Bitcoin Core wallet cannot make ordinal-aware transactions. Making
+ordinal-aware transactions requires [`ord`](https://github.com/casey/ord), the
+ordinal theory utility. `ord wallet` subcommands wrap an existing Bitcoin Core
+wallet.
+
+This guide contains everything you need to know about creating, sending, and
+receiving inscriptions:
+
+1. Installing Bitcoin Core
+2. Syncing the Bitcoin blockchain
+3. Creating a Bitcoin Core wallet
+4. Using `ord wallet receive` to receive satoshis
+5. Creating inscriptions with `ord wallet inscribe`
+6. Sending inscriptions with `ord wallet send`
+7. Receiving inscriptions with `ord wallet receive`
+
+Getting Help
+------------
+
+A guide can only do so much! If you get stuck, try asking for help on the
+[Ordinal Theory Discord Server](https://discord.com/invite/87cjuz4FYg), or
+checking GitHub for relevant [issues](https://github.com/casey/ord/issues) and
+[discussions](https://github.com/casey/ord/discussions).
+
+Installing Bitcoin Core
+-----------------------
+
+Bitcoin Core is available from [bitcoincore.org](https://bitcoincore.org/) on
+the [download page](https://bitcoincore.org/en/download/).
+
+This guide does not cover installing Bitcoin Core in detail. Once Bitcoin Core
+is installed, you should be able to run `bitcoind -version` successfully from
+the command line.
+
+Configuring Bitcoin Core to use Signet
+--------------------------------------
+
+`ord wallet` subcommands cannot yet be used on mainnet, so your Bitcoin Core
+node must be configured to use another chain. This guide uses signet, but
+testnet or regtest mode may also be used.
+
+To configure your Bitcoin Core node to use signet, add `signet=1` to your
+`bitcoin.conf` configuration file, or pass `-signet` to `bitcoind` and
+`bitcoin-cli`, e.g. `bitcoind -signet` and `bitcoin-cli -signet`.
+
```

Comment:
Add paragraph specifying the need for building the transaction index.
Add `txindex=1` to `bitcoin.conf` or `bitcoind -txindex` (`bitcoin-cli` does not need that option).